src/app/data.ts

```diff
@@ -0,0 +1,393 @@
+import { TimelineEvent } from 'ngx-d3timeline';
+
+export const data: TimelineEvent[] = [
```

Comment:
maybe we call this something like deliveryData?

Then here and in component its more meaningful ie:
`export class AppComponent {
  title = 'ngx-d3timeline';  	  
  data = deliveryData;  
}	}`

Not overly fussed by it though.

Reply:
done